### ISSUE TITLE
Outdated notice on hugo's theme site

### ISSUE THREAD

**Issue by lukaswe** [dewittn/hugo-html5up-alpha#6]
Hello @dewittn , 

On Hugo's Theme site, this theme got an outdated notice.
As far as I can see, this template works with hugo exteded 0.99. 

I think, youre theme would be more attractive, if you could accomodate hugo's reqest in order to remove that box:
![image](https://user-images.githubusercontent.com/24406480/173248998-1ce0084a-0b50-4e99-a6d1-a48db15fecc8.png)

**Comment by dewittn** [comment]
I've been working on an update to the theme but just haven't had time to publish it yet. But thanks for the heads up!

Hopefully I'll get that update out soon!

**Comment by lukaswe** [comment]
Glad to hear, @dewittn ! Looking forward to it.
I've made some adjustments to the theme in my page draft, including:

- Shortcodes for images with caption
- added a 404 page
- and removed google from fonts hosting.
I am not a developer so sending a pull request is rather low on my list and regards some question marks. But would these things be interesting for you?

**Comment by dewittn** [comment]
@lukaswe sure, I would be happy to take a look at your changes. Do you have the code ghosted on GitHub or any other public repository I can take a look?

**Comment by lukaswe** [comment]
no, unfortunately, I have the code only on a local git. I'll see, if a 
pull request might be sufficient. I don't know, if I can find the time 
next week.

Am Do, 16. Jun 2022 um 10:19:02 -0700 schrieb dewittn 
***@***.***>:
